src/Console/ServeCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace TightenCo\Jigsaw\Console;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ServeCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('serve')
+            ->setDescription('Serve local site with php built-in server.')
+            ->addArgument(
```

Comment:
Do you think I should add host name as an option while i'm at it? just so it's there? 
`jigsaw serve production --host=mysite.dev --port=8080` No sure it will be used a lot... thoughts?